crates/hc_seed_bundle/Cargo.toml

```diff
@@ -14,7 +14,7 @@ edition = "2018"
 futures = "0.3.28"
 one_err = "0.0.8"
 rmp-serde = "1.1.1"
-rmpv = { version = "1.0.0", features = [ "with-serde" ] }
+rmpv = { version = "=1.0.0", features = [ "with-serde" ] } # Version 1.0.1 requires Rust 1.70.0
```

Comment:
I feel like this is the wrong place to address this problem. Any other projects, whether they be holochain or something holo-adjacent can pin rmpv in ***their*** Cargo.toml to be able to build on an older rust version.

Reply:
Equally, many projects declare an MSRV and test from that version. We can stick to `stable` as our supported version but I think that's going to be a pain alongside Holochain and Holonix. This is something we can revisit though, 

I've replaced this PR with #115 because the project currently doesn't build on `stable` either. I've also bumped to `0.3.0` because this is actually a change that is visible in the API to rename the `traits` modules.

Reply:
Actually, I've thought about this one again. People can lock the API downstream but they cannot lock binary dependencies, so we are putting a hard dependency on Rust 1.70+ for `lair_keystore`. That is going for force us to upgrade our Rust version in Holochain to 1.70